### DISCUSSION TITLE
Logo module position selection in Astroid Manager

### DISCUSSION THREAD

**Discussion by Banuk1616** [templaza/astroid-framework#727] [Ideas]
My English is not so good and I don't know if I can write my suggestion in English. I hope it is still understandable what I mean.

I have a logo that is not positioned at the normal position of the logo in the Astroid header.
I have solved this with a separate module and it works fine. But it would be easier if you could select a module position in the template manager under logo options and use the native logo options. This would make it unnecessary to create a separate module for the logo.

**Comment by Chacapamac** [comment]
I second, this is a good idea.

**Comment by Banuk1616** [comment]
Hello Sonny, I’m currently working on a design/layout for a private website where the logo should not be displayed in one of the predefined positions in the Astroid header. This reminded me that I had opened a thread about this topic a while ago. Chacapamac thought my idea was good back then, and today I realized again that it would be a small but helpful improvement if the logo position could be selected within the logo options. Could you perhaps consider whether it makes sense to make this possible in a future version of the Astroid Framework?

Best regards Olli
